modules/administration_manual/pages/release_notes.adoc

```diff
@@ -17,6 +18,64 @@
 * xref:changes-in-8.0[Changes in 8.0]
 * xref:changes-in-7.0[Changes in 7.0]
 
+[[changes-in-10.1]]
+
```

Comment:
there is no need for a blank line (22), you can remove it

Reply:
done.